apps/src/templates/instructions/RubricField.jsx

```diff
@@ -118,5 +118,5 @@ class RubricField extends Component {
     );
   }
 }
-
+export const UnconnectedRubricField = RubricField;
```

Comment:
Naming convention: I've started using `Unwrapped` instead of `Unconnected` specifically for cases like this where the component isn't "connected" to a redux store in the [`react-redux`](https://react-redux.js.org/) sense, only wrapped by Radium.